src/main/java/me/xmrvizzy/skyblocker/config/categories/GeneralCategory.java

```diff
@@ -204,6 +204,19 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
 								.build())
 						.build())
 				
+				//Item Cooldown
+				.group(OptionGroup.createBuilder()
+						.name(Text.translatable("text.autoconfig.skyblocker.option.general.itemCooldown"))
+						.collapsed(true)
+						.option(Option.<Boolean>createBuilder()
+								.name(Text.translatable("text.autoconfig.skyblocker.option.general.itemCooldown.enableItemCooldowns"))
+								.binding(defaults.general.itemCooldown.enableItemCooldowns,
+										() -> config.general.itemCooldown.enableItemCooldowns,
+										newValue -> config.general.itemCooldown.enableItemCooldowns = newValue)
+								.controller(ConfigUtils::createBooleanController)
+								.build())
+						.build())
+				
```

Comment:
This line has unnecessary trailing whitespace (doesn't seem to be visible on GitHub).

Reply:
fixed